controllers/discordactions.js

```diff
@@ -67,25 +66,19 @@ const createGroupRole = async (req, res) => {
 const getAllGroupRoles = async (req, res) => {
   try {
     const { groups } = await discordRolesModel.getAllGroupRoles();
-    const groupsWithMemberCount = await discordRolesModel.getNumberOfMemberForGroups(groups);
-    const groupCreatorIds = groupsWithMemberCount.reduce((ids, group) => {
-      ids.add(group.createdBy);
-      return ids;
-    }, new Set());
-    const groupCreatorsDetails = await retrieveUsers({ userIds: Array.from(groupCreatorIds) });
-    const groupsWithUserDetails = groupsWithMemberCount.map((group) => {
-      const groupCreator = groupCreatorsDetails[group.createdBy];
-      return {
-        ...group,
-        firstName: groupCreator.first_name,
-        lastName: groupCreator.last_name,
-        image: groupCreator.picture?.url,
-      };
-    });
-
+    const dev = req.query.dev === "true";
```

Comment:
You are only sending dev flag as a query parameter??
Where is rough API contract??

Reply:
dev is feature flag. 
I will see where I can create a API contract

Reply:
Hey @shubhamsigdar1 , 
I have added the API contract for this API in the below PR. 
Please review.
https://github.com/Real-Dev-Squad/website-api-contracts/pull/152